lib/engine/game/g_1880/step/company_pending_par.rb

```diff
@@ -7,18 +7,61 @@ module Game
     module G1880
       module Step
         class CompanyPendingPar < Engine::Step::CompanyPendingPar
```

Comment:
Are the changes to this step still relevant after my parring changes? Or is there something I should review in it?

Reply:
probably not, this is for the abc/bcd choice for BCR.